src/main/java/automation/db/UserData.java

```diff
@@ -0,0 +1,127 @@
+package automation.db;
+
+import automation.Session;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+
+public class UserData {
+    static public class User
+    {
+        final public int id;
+        final public String first_name;
+        final public String last_name;
+        final public String login;
+        final public String password_hash;
+        final public String date_of_birth;
+
+        public User(int id,
+                    String first_name,
+                    String last_name,
+                    String login,
+                    String password_hash,
+                    String date_of_birth
+                    ) {
+            this.id = id;
+            this.first_name = first_name;
+            this.last_name = last_name;
+            this.login = login;
+            this.password_hash = password_hash;
+            this.date_of_birth = date_of_birth;
+        }
+    }
```

Comment:
You decided to use User class but its constructor is very heavy, suggest to create builder for User class